### ISSUE TITLE
Username-select Review updated design on Zeplin. If the user enters a username that has already been taken they should get an ALERT when “Next” is clicked. The “Next” button in the upper right hand corner will have reduced opacity of 50% and not be clickable until a user has entered text into the input field.

### ISSUE THREAD

**Issue by metzgerenator** [OpenMic/OpenMic---iOS#12]

**Comment by metzgerenator** [comment]
@adenaro I would need to make a separate api call each time for this.  I'm not sure if you have an endpoint for this and if it would be health for your backend.   @aaron13michael Can you advise on this?

**Comment by adenaro** [comment]
@metzgerenator Is this still a blocker?

**Comment by metzgerenator** [comment]
@adenaro I'm not blocked on this.   Check out the dialog on the next build and let me know if you want the wording changed

**Comment by metzgerenator** [comment]
@adenaro I'm going to have to come back to adding the opacity logic.  I'm having some issues with switching out the button.  It should be minor, but I'd rather come back to something aesthetic like this after I have all the api calls working.

**Comment by adenaro** [comment]
@metzgerenator Please edit the task to make opacity its own issue and move it to blocker.
